src/lua/lua_cryptobox.c

```diff
@@ -1452,7 +1458,10 @@ lua_cryptobox_hash_finish(struct rspamd_lua_cryptobox_hash *h)
 		memcpy(h->out, out, ssl_outlen);
 		break;
 	case LUA_CRYPTOBOX_HASH_HMAC:
-		HMAC_Final(h->content.hmac_c, out, &ssl_outlen);
+		size_t ssl_outlen_size_t = ssl_outlen;
```

Comment:
Why do we have this extra variable?

Reply:
Because `EVP_MAC_final` has a `size_t* outl` argument while `ssl_outlen` is `unsigned int`. That is why I am converting it to the size_t using temporary variable.

Reply:
Yes, but maybe it's betted to have `ssl_outlen` to be `size_t` then?

Reply:
Yes, it will be probably be better. Thank you!